### PR TITLE
Fix doc validation test errors

### PR DESCRIPTION
**Pull request by vkovec**

### CODE REVIEW

**Review by windxu88 — APPROVED** [review]
API docs validation test passes with the doc changes:
https://unity-ci.cds.internal.unity3d.com/job/16420047/dependency-graph

File: com.unity.formats.fbx/Editor/ExportSettings.cs [hunk 1]
```diff
@@ -8,42 +8,88 @@
...
+        /// </summary>
+        LocalCentered = 0,
+        /// <summary>
+        /// Uses the world position of the GameObjects.
+        /// </summary>
+        WorldAbsolute = 1,
+        /// <summary>
+        /// Exports the object to (0,0,0).
+        /// </summary>
+        Reset = 2 /* For convert to model only, no UI option*/
```
Comment (benoithudson) [inline com.unity.formats.fbx/Editor/ExportSettings.cs:66]: I think this comment should be in the Reset summary doc.

**Review by benoithudson — COMMENTED** [review]
Works for me. Probably Sebastien should review the new doc text?

**Review by benoithudson — APPROVED** [review]

File: com.unity.formats.fbx/Editor/ExportSettings.cs (hunk 1 above)
Comment (vkovec) [reply com.unity.formats.fbx/Editor/ExportSettings.cs]: done

File: com.unity.formats.fbx/Editor/ExportSettings.cs [hunk 2]
```diff
@@ -8,42 +8,89 @@
...
+        /// If you select multiple GameObjects for export, the FBX Exporter centers GameObjects
+        /// around a shared root while keeping their relative placement unchanged.
+        /// </summary>
+        LocalCentered = 0,
+        /// <summary>
+        /// Uses the world position of the GameObjects.
+        /// </summary>
+        WorldAbsolute = 1,
+        /// <summary>
+        /// Exports the object to (0,0,0).
```
Comment (sebastienduverne) [inline com.unity.formats.fbx/Editor/ExportSettings.cs:64]:
```suggestion
        /// Exports the GameObject to (0,0,0).
```

File: com.unity.formats.fbx/Editor/ExportSettings.cs [hunk 3]
```diff
@@ -8,42 +8,89 @@
...
     /// </summary>
-    public enum Include { Model = 1, Anim = 2, ModelAndAnim = 3 }
+    public enum Include
+    {
+        /// <summary>
+        /// Export the model without animation.
+        /// </summary>
+        Model = 1,
+        /// <summary>
+        /// Export animation only.
```
Comment (sebastienduverne) [inline com.unity.formats.fbx/Editor/ExportSettings.cs:39]:
```suggestion
        /// Export the animation only.
```

**Review by sebastienduverne — APPROVED** [review]
Very minor things.

**Pull request merged**